docs/markdown/Using Pants/using-pants-in-ci.md

```diff
@@ -196,6 +196,43 @@ args = ["-vv", "--no-header"]
 
 Then, in your CI script or config, set the environment variable `PANTS_CONFIG_FILES=pants.ci.toml` to use this new config file, in addition to `pants.toml`.
 
+Using with pre-commit
+---------------------
```

Comment:
This means H2, and it makes the H3 for "### Tuning resource consumption (advanced)" in the wrong place. Maybe keep it as H2 but move to lower in the file?